server/models/messages.js

```diff
@@ -1,7 +1,7 @@
 const {HttpError} = require('../utils/errors');
 const prisma = require('../db');
 class Message {
- constructor(userId,message,groupId){
+ constructor(userId,groupId,message=""){
```

Comment:
The implementation of having an optional message argument for the constructor  in this class , gives us the flexibility of using the view function without  any unecessary modifications to the class(keeping it DRY),  and also message input validation is part of the route implementation for post group messages, so we never have to worry if the message argument has been provided to the constructor or if it will ever be empty under any other scenario.